mm2src/coins/eth/v2_activation.rs

```diff
@@ -217,11 +302,57 @@ impl EthCoin {
             logs_block_range: self.logs_block_range,
             nonce_lock: self.nonce_lock.clone(),
             erc20_tokens_infos: Default::default(),
+            nfts_infos: Default::default(),
             abortable_system,
         };
 
         Ok(EthCoin(Arc::new(token)))
     }
+
+    /// Creates a global NFT instance from a platform coin, such as Ethereum.
```

Comment:
btw could you add a description what 'global' means here?

Reply:
Updated doc com here, provided more Global NFT description

Reply:
Global means its a global (common) instance for all nfts owned by user (related to specific blockchain type). So we dont need to create EthCoin for each user's non fungible token.